oss-cloud-frontend/src/components/ContributionsPage.vue

```diff
@@ -77,10 +139,25 @@
 <script>
 import { loadContributionsAxios } from "./../axiosService.js";
 import { updateContributionStatus } from "./../axiosService.js";
+const moment = require("moment");
+const dateFormat = "DD/MM/YYYY";
 
 export default {
   data() {
     return {
+      searchText: "",
+      statusFilter: "All",
+      usernameSearch: false,
+      titleSearch: false,
+      repoSearch: false,
+      menu1: false,
+      menu2: false,
+      dateFrom: moment("2000-01-01 01:00")
+        .toISOString()
+        .substr(0, 10),

```

Comment:
Why we still use substr? Can we use format to set right value?

Reply:
Vuetify's date picker component requires date as first 10 characters of ISO formatted date string, as can be seen on this [example](https://vuetifyjs.com/en/components/date-pickers#date-pickers-formatting-date).